### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by brunopulis**
Adding Awesome Acessibility

### CODE REVIEW

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/awesome/blob/master/contributing.md

File: readme.md [hunk 1]
```diff
@@ -52,6 +52,7 @@
 - [JavaScript Must Watch Talks](https://github.com/bolshchikov/js-must-watch)
 - [ES6 Tools](https://github.com/addyosmani/es6-tools)
 - [Web Performance Optimization](https://github.com/davidsonfellipe/awesome-wpo)
+- [Web Acessibility](https://github.com/brunopulis/awesome-a11y)
```
Comment (sindresorhus) [inline readme.md]: Add it to the end.

**Comment by sindresorhus** [comment]
Could use some more content too. And descriptions with the links.

**Comment by sindresorhus** [comment]
Added. Would be awesome if you could link back here in your list so users can easily discover more awesome lists ;)

**Comment by brunopulis** [comment]
@sindresorhus Yeah that's a great ideia, I link back your list :D 
About use more content I'll include thanks for the feedback

**Pull request closed without merging**